initial reports

graza-io opened the pull request.

graza-io commented:
Note: This still requires a `require` section once we confirm version of plugin.

misraved commented:
@graza-io please take a look at the following review comments:

 - `GitHub Branch Activity Report` 
   - waited for close to 10-15 mins, however, it was stuck in the loading state. Ultimately I get a timeout issue.
   - Error:
```
 Error
ERROR: rpc error: code = Unknown desc = non-200 OK status code: 502 Bad Gateway body: "{\n \"data\": null,\n \"errors\":[\n {\n \"message\":\"Something went wrong while executing your query. This may be the result of a timeout, or it could be a GitHub bug. Please include `D682:1FB9:87F25:95C61:64AF11AE` when reporting this issue.\"\n }\n ]\n}\n" (SQLSTATE HV000)
```

 - `GitHub Open Issue Age Report` 
    - I think the issue format should stay true to the github issue format. You could use https://github.com/turbot/steampipe-utils/blob/main/scripts/slack_message.sh#L34 for more inspiration.

- `GitHub Organization 2FA Report`
    - All the report columns and the titles should be updarted to use `2FA` instead of `2fa`
    - What does Unknown/null refer to?
- `GitHub Organization Member Privileges Report`
    - What is the difference between `null` and `none` values of the `Base Permission` column?

graza-io commented:
Adjusted issue/pr column to be title then number as per GitHub [0804228](https://github.com/turbot/steampipe-mod-github-insights/pull/4/commits/0804228be0b202487d94c56ac3d69a1065937f30)

Adjusted 2fa to 2FA [099915b](https://github.com/turbot/steampipe-mod-github-insights/pull/4/commits/099915b7dd4df0ad5c3f3a77b91483deddfc2a86)

graza-io commented:
@misraved - I haven't done a full comparison against the AWS ones but the files do exist: [README](https://github.com/turbot/steampipe-mod-github-insights/blob/initial-reports/README.md) & [docs/index](https://github.com/turbot/steampipe-mod-github-insights/blob/initial-reports/docs/index.md)

misraved commented:
My bad @graza-io, I missed them. Thanks for adding those docs in 👍.
